sim/src/mechanics/intersection.rs

```diff
@@ -254,24 +254,53 @@ impl IntersectionSimState {
         map: &Map,
         scheduler: &mut Scheduler,
     ) {
+        // trivial function that returns true if the stage is just crosswalks
+        // todo bb: consider moving to Stage impl
```

Comment:
Let's resolve the todo, then I'll update.

Reply:
Sorry, I was on mobile earlier. Yeah, this can live in Stage, with a docstring like `/// True if the stage only consists of protected crosswalks.`

Calling it `is_all_walk` is _slightly_ misleading. Somebody could manually make a stage that only has 2 protected crosswalks, but is missing a crosswalk (and included in some other stage instead). But I think the name is fine.